src/wasm-interpreter.h

```diff
@@ -1273,7 +1273,7 @@ class ExpressionRunner : public OverriddenVisitor<SubType, Flow> {
     for (auto item : arguments) {
       exn->values.push_back(item);
     }
-    throwException(Literal(std::move(exn)));
+    throwException(Literal::makeExnref(std::move(exn)));
```

Comment:
It looks like this line didn't have to change.

Reply:
Yeah, just changed it for clarity, since 16 lines above we have

```cpp
  Flow visitRefFunc(RefFunc* curr) {
    NOTE_ENTER("RefFunc");
    NOTE_NAME(curr->func);
    return Literal::makeFuncref(curr->func); // <-
  }
```

Reply:
In fact, while `Literal::makeExnref` exists, it isn't even used at all otherwise. And this seemed like the perfect place to make use of it.